src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3464,28 +3627,238 @@ void reportUnsafeIfNeeded(MemberResolutionResult<TMember> methodResult, BindingD
                     //CONSIDER: Return a bad expression so that HasErrors is true?
                 }
             }
-        }
 
-        private static ParameterSymbol GetCorrespondingParameter(ref MemberAnalysisResult result, ImmutableArray<ParameterSymbol> parameters, int arg)
-        {
-            int paramNum = result.ParameterFromArgument(arg);
-            return parameters[paramNum];
-        }
-#nullable disable
-
-        private static TypeWithAnnotations GetCorrespondingParameterTypeWithAnnotations(ref MemberAnalysisResult result, ImmutableArray<ParameterSymbol> parameters, int arg)
-        {
-            int paramNum = result.ParameterFromArgument(arg);
-
-            if (paramNum == parameters.Length - 1 && result.Kind == MemberResolutionKind.ApplicableInExpandedForm)
+            static ParameterSymbol getCorrespondingParameter(in MemberAnalysisResult result, ImmutableArray<ParameterSymbol> parameters, int arg)
             {
-                Debug.Assert(result.ParamsElementTypeOpt.HasType);
-                Debug.Assert(result.ParamsElementTypeOpt.Type != (object)ErrorTypeSymbol.EmptyParamsCollectionElementTypeSentinel);
-                return result.ParamsElementTypeOpt;
+                int paramNum = result.ParameterFromArgument(arg);
+                return parameters[paramNum];
             }
 
-            return parameters[paramNum].TypeWithAnnotations;
+            BoundExpression bindInterpolatedStringHandlerInMemberCall(
```

Comment:
>bindInterpolatedStringHandlerInMemberCall [](http://example.com/codeflow?start=28&length=41)

This used to be ```BindInterpolatedStringHandlerInMemberCall``` at the end of the `Binder_InterpolatedString.cs`. No meaningful changes during the move.